cinema/views.py

```diff
@@ -21,7 +24,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
 )
```

Comment:
Code Style: It's recommended to add a blank line at the end of all your files. This is a common convention that helps certain tools read and process your code correctly.